core/src/test/java/org/infinispan/jmx/CacheMBeanTest.java

```diff
@@ -119,16 +121,6 @@ public void testDuplicateJmxDomainOnlyCacheExposesJmxStatistics() throws Excepti
       }
    }
 
-   public void testMalformedCacheName(Method m) throws Exception {
```

Comment:
TBH that test is just wrong: it does not use JMX to interact with the cache and it is just attempting to retrieve a cache from a container using a weird name... and it passes because it throws no exception since it creates the cache on the fly...